sdks/python/apache_beam/coders/coders_test_common.py

```diff
@@ -667,11 +667,13 @@ def test_state_backed_iterable_coder(self):
     state = {}
 
     def iterable_state_write(values, element_coder_impl):
+      global state
```

Comment:
what is the testing plan for this change? I think we need to find a way to run some tests on both picklers, perhaps similarly to how we parameterize FnAPI runner tests, or change default pickler in one of the tox configurations.

Reply:
I think we need more thorough E2E tests. This current PR only has unit tests for cloudpickler (the new thing introduces) to make sure that the defaults work.

I would like some more complete/complex tests, but would love for them to be in a separate PR.